modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/TBlat_conf.pm

```diff
@@ -19,11 +19,17 @@ limitations under the License.
 
 =head1 NAME
 
-Bio::EnsEMBL::Compara::PipeConfig::EBI::TBlat_conf
+Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::TBlat_conf
+
+=head1 SYNOPSIS
+
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::TBlat_conf -host mysql-ens-compara-prod-X -port XXXX \
+        -mlss_id_list "[9769,9784,9754,9770,9785,9786,9771,9755,9767,9768,9782,9783,9756,9752,9753]" \
+        -pipeline_name ${COMPARA_DIV}_tblat_batchX_${CURR_ENSEMBL_RELEASE}
```

Comment:
Use instead `-mlss_id 591 -ref_species mus_musculus`. Or `-mlss_id 574 -ref_species danio_rerio` like in the parent class